src/ext/themelist.js

```diff
@@ -48,7 +49,8 @@ var themeData = [
     ["Tomorrow Night 80s"   ,"tomorrow_night_eighties" ,  "dark"],
     ["Twilight"             ,"twilight"                ,  "dark"],
     ["Vibrant Ink"          ,"vibrant_ink"             ,  "dark"],
-    ["GitHub Dark"          ,"github_dark"             ,  "dark"]
+    ["GitHub Dark"          ,"github_dark"             ,  "dark"],
+    ["CloudEditor Dark", "cloud_editor_dark", "dark"]
```

Comment:
**nitpick:** Similar comment as above: could you use the same formatting convention that's used for others? I.e., aligning indexes vertically